app/polymer/package.json

```diff
@@ -53,12 +53,14 @@
     "webpack-hot-middleware": "^2.22.0"
   },
   "devDependencies": {
+    "lit-html": "^0.9.0",
```

Comment:
Why do you need the "lit-html" and "polymer-webpack-loader" in devDeps ?

Reply:
For `lit-html`, it's probably because you need a single instance of that package for `instanceof TemplateResult` to pass